### PR TITLE
Add some doc about billing

### PR DESCRIPTION
**Pull request by julien-c**

### CODE REVIEW

File: docs/hub/billing.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Billing
+
+At Hugging Face we build a collaboration platform for the ML community (i.e. the Hub), and we **monetize on providing simple access to compute for AI**, directly from the collaboration platform.
```
Comment (osanseviero) [inline docs/hub/billing.md:3]:
```suggestion
At Hugging Face, we build a collaboration platform for the ML community (i.e., the Hub), and we **monetize by providing simple access to compute for AI**, directly from the collaboration platform.
```

File: docs/hub/billing.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+We partner with cloud providers like AWS and Azure, and we also build our own self-serve compute services.
```
Comment (osanseviero) [inline docs/hub/billing.md:5]:
```suggestion
We partner with cloud providers like AWS and Azure and build our own self-serve compute services.
```

File: docs/hub/billing.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+Both users and organizations accounts have a billing system set up. You can submit your payment info then access "pay-as-you-go" services.
```
Comment (osanseviero) [inline docs/hub/billing.md:7]:
```suggestion
Both users and organizations accounts have a billing system set up. You can submit your payment info and then access "pay-as-you-go" services.
```

**Review by osanseviero — APPROVED** [review]
Thanks!

File: docs/hub/billing.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Billing
+
+At Hugging Face, we build a collaboration platform for the ML community (i.e., the Hub), and we **monetize by providing simple access to compute for AI**, directly from the collaboration platform.
+
+We partner with cloud providers like AWS and Azure and build our own self-serve compute services.
```
Comment (Michellehbn) [inline docs/hub/billing.md:5]: Azure, and

File: docs/hub/billing.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+Both users and organizations accounts have a billing system set up. You can submit your payment info and then access "pay-as-you-go" services.
```
Comment (Michellehbn) [inline docs/hub/billing.md:7]: All user and organization accounts

**Review by Michellehbn — COMMENTED** [review]
2 teensy grammar recommendations! Looks great!

File: docs/hub/billing.md (hunk 4 above)
Comment (julien-c) [inline docs/hub/billing.md:5]:
```suggestion
We partner with cloud providers like AWS and Azure, and build our own self-serve compute services.
```

File: docs/hub/billing.md (hunk 5 above)
Comment (julien-c) [inline docs/hub/billing.md:7]:
```suggestion
All users and organizations accounts have a billing system set up. You can submit your payment info and then access "pay-as-you-go" services.
```

File: docs/hub/billing.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# Billing
+
+At Hugging Face, we build a collaboration platform for the ML community (i.e., the Hub), and we **monetize by providing simple access to compute for AI**, directly from the collaboration platform.
+
+We partner with cloud providers like AWS and Azure, and build our own self-serve compute services.
```
Comment (jeffboudier) [inline docs/hub/billing.md:5]:
```suggestion
We also partner with cloud providers like AWS and Azure to make it easy for customers to use Hugging Face directly in their cloud of choice. These solutions and usage are billed directly by the cloud provider. Ultimately we want people to be able to have great options to use Hugging Face wherever they build Machine Learning.
```

File: docs/hub/billing.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
+# Billing
+
+At Hugging Face, we build a collaboration platform for the ML community (i.e., the Hub), and we **monetize by providing simple access to compute for AI**, directly from the collaboration platform.
```
Comment (jeffboudier) [inline docs/hub/billing.md:3]:
```suggestion
At Hugging Face, we build a collaboration platform for the ML community (i.e., the Hub), and we **monetize by providing simple access to compute for AI**, with services like AutoTrain, Spaces and Inference Endpoints, directly accessible from the Hub, and billed by Hugging Face to the credit card on file.
```

**Comment by julien-c** [comment]
will merge now but we can always open another PR later to refine/expand

**Pull request merged**